src/main/java/com/WebController/EventController.java

```diff
@@ -63,4 +81,53 @@ public String receivePatient (@RequestBody List<Event> events){
         return list;
     }
 
+    @PostMapping(value = "/userregister", consumes = "application/json")
+    @ResponseBody
+    public String userRegistration(@RequestBody User user){
+        if (this.userRepository.findUserByEmailAddressIgnoreCase(user.getEmailAddress())!=null){
+            return "user already exists";
```

Comment:
Don't return Strings like this as a REST API. The consumer of this API may not be a human.